custom_components/gree/climate.py

```diff
@@ -318,6 +323,7 @@ def FetchResult(self, cipher, ip_addr, port, timeout, json):
         clientSock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         clientSock.settimeout(timeout)
         clientSock.sendto(bytes(json, "utf-8"), (ip_addr, port))
+        _LOGGER.info('3')
```

Comment:
Why do we need this line?

Reply:
My bad, we do not, seems I did not read my own pull request well enough. This was some leftover debugging code from some previous codebase or something, no need for it. Amended my pull request and removed it.